redis/endpoint.bal

```diff
@@ -95,7 +101,7 @@ public client class Client {
     # + return - Size of the string stored in the destination key, that is equal to the size of the longest input
```

Comment:
The size of the string stored in the destination key, which is equal to the size of the longest input string, or `error` in case an error occurs